File: test/EntityFramework/VBTests/_VbAdvancedPatterns/Partials/AdvancedPatternsModelFirstContext.vb
'---------------------------------------------------------------------
' <copyright file="AdvancedPatternsModelFirstContext.vb" company="Microsoft">
'      Copyright (c) Microsoft Corporation.  All rights reserved.
' </copyright>
'
' @owner       avickers
' @backupOwner bricelam
'---------------------------------------------------------------------

Namespace AdvancedPatternsVB

    Partial Friend Class AdvancedPatternsModelFirstContext

        Public Sub New(ByVal nameOrConnectionString As String)
            MyBase.New(nameOrConnectionString)
            MyBase.Configuration.LazyLoadingEnabled = False
        End Sub

    End Class

End Namespace

File: test/EntityFramework/FunctionalTests/TestModels/TemplateModels/CsAdvancedPatterns/Partials/AdvancedPatternsModelFirstContext.cs
namespace FunctionalTests.ProductivityApi.TemplateModels.CsAdvancedPatterns
{
    internal partial class AdvancedPatternsModelFirstContext
    {
        public AdvancedPatternsModelFirstContext(string nameOrConnectionString)
            : base(nameOrConnectionString)
        {
            Configuration.LazyLoadingEnabled = false;
        }
    }
}